shared/src/popup.js

```diff
@@ -30,6 +33,23 @@ function setStatus(type) {
 
   switch (type) {
     case 'no_session': {
+      const openKagiElement = document.querySelector('#open_kagi');
+      const permission = {
+        origins: ['https://kagi.com/'],
```

Comment:
I'd recommend making this `'https://kagi.com/*'` so it's an exact match to the host listed in the manifest as of the change I made in https://github.com/kagisearch/browser_extensions/pull/15/commits/191af063301ec46cef3318e5333d1f47915c6559

Reply:
Thanks, fixed! I looked over your PR before rebasing onto it, but missed that :)